src/main/scala/com/bwsw/cloudstack/entities/common/JsonMapper.scala

```diff
@@ -27,6 +27,9 @@ import com.fasterxml.jackson.databind.{DeserializationFeature, ObjectMapper}
 import com.fasterxml.jackson.module.scala.DefaultScalaModule
 import org.slf4j.LoggerFactory
 
+/**
+  * Class is responsible for conversion from entities to json string and from json string to entities
+  */
```

Comment:
This mapper provides functionality to convert your model objects (classes and structs) to and from json string.

Reply:
ok